bundles/org.openhab.binding.bluetooth.hdpowerview/src/main/java/org/openhab/binding/bluetooth/hdpowerview/internal/shade/ShadeHandler.java

```diff
@@ -0,0 +1,533 @@
+        if (command instanceof StopMoveType stopMove) {
+            if (StopMoveType.STOP == stopMove) {
+                switch (channelUID.getId()) {
+                    case CHANNEL_SHADE_PRIMARY:
+                        command = dataReader.getPrimary();
+                        break;
+                    case CHANNEL_SHADE_SECONDARY:
+                        command = dataReader.getSecondary();
+                        break;
+                    case CHANNEL_SHADE_TILT:
+                        command = dataReader.getTilt();
+                        break;
+                }
+            }
+        }
+
+        // convert up/down commands to position command
+        if (command instanceof UpDownType updown) {
+            command = UpDownType.DOWN == updown ? PercentType.ZERO : PercentType.HUNDRED;
+        }
+
+        if (command instanceof PercentType percent) {
+            Capabilities capabilities = this.capabilities;
+            if (capabilities == null) {
+                return;
+            }
+
+            try {
+                switch (channelUID.getId()) {
+                    case CHANNEL_SHADE_PRIMARY:
+                        if (capabilities.supportsPrimary()) {
+                            scheduleWritePosition(new ShadeDataWriter().withSequence(writeSequence++)
+                                    .withPrimary(percent.doubleValue()).getEncrypted(getEncryptionKey()));
+                        }
+                        break;
+
+                    case CHANNEL_SHADE_SECONDARY:
+                        if (capabilities.supportsSecondary()) {
+                            scheduleWritePosition(new ShadeDataWriter().withSequence(writeSequence++)
+                                    .withSecondary(percent.doubleValue()).getEncrypted(getEncryptionKey()));
+                        }
+                        break;
+
+                    case CHANNEL_SHADE_TILT:
+                        if (capabilities.supportsTiltOnClosed() || capabilities.supportsTilt180()
+                                || capabilities.supportsTiltAnywhere()) {
+                            scheduleWritePosition(new ShadeDataWriter().withSequence(writeSequence++)
+                                    .withTilt(percent.doubleValue()).getEncrypted(getEncryptionKey()));
+                        }
+                        break;
+                }
+            } catch (InvalidKeyException | IllegalArgumentException | NoSuchAlgorithmException | NoSuchPaddingException
+                    | InvalidAlgorithmParameterException | IllegalBlockSizeException | BadPaddingException e) {
+                logger.warn("handleCommand() device={} error={}", device.getAddress(), e.getMessage(),
+                        logger.isDebugEnabled() ? e : null);
+            }
+        }
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        configuration = getConfigAs(ShadeConfiguration.class);
```

Comment:
```suggestion
        updateStatus(ThingStatus.UNKNOWN);
        configuration = getConfigAs(ShadeConfiguration.class);
```

Reply:
That is not necessary. The inherited super method does already set the thing status to UNKNOWN.